docbook/Developers_Guide/en-US/Events_Reference_Bug.xml

```diff
@@ -516,6 +516,38 @@
 				</itemizedlist>
 			</blockquote>
 		</blockquote>
+                
+                <blockquote id="dev.eventref.bug.action.fileadded">
+			<title>EVENT_FILE_ADDED (Execute)</title>
+
+			<blockquote>
+				<para>
+					This event allows plugins to do post-processing of uploaded file.
+				</para>
+
+				<itemizedlist>
+					<title>Parameters</title>
+					<listitem><para>&lt;Array of Array&gt;: File ID, name db table</para></listitem>
```

Comment:
Not sure this documentation is understandable.

- I think the first array is abstracted away by the plugin system.  So the parameter is an associative array with two keys in it.  Mention the key names and description for each one.
- I wouldn't refer to table names, but refer to expected values.

Reply:
Fixed